pegasus/sites.v3/hourofcode.com/views/resources_nav.haml

```diff
@@ -0,0 +1,13 @@
+%div{style: "margin-top: 10px;"}
+  %a{href: resolve_url('/promote')}
+    %div.display-link
+      %img.display-img{src: "/images/flyers.png"}
+      %div.link-text.display-purple= hoc_s(:resources_nav_promote)
+  %a{href: resolve_url('https://code.org/learn')}
+    %div.display-link
+      %img.display-img{src: "/images/try_tutorials.png"}
```

Comment:
I haven't done a blanket change for all of hoc.com, but if you touch files could you check that all links, files, and images are localized?

    src: localized_image("/images/try_tutorials.png"}